src/main/frontend/fs/sync.cljs

```diff
@@ -1544,8 +1544,7 @@
   [type {:keys [dir path _content stat] :as _payload}]
   (when-let [current-graph (state/get-current-repo)]
     (when (string/ends-with? current-graph dir)
-      (when-not (some-> (state/get-file-sync-state current-graph)
-                        sync-state--stopped?)
+      (when-not (sync-state--stopped? (state/get-file-sync-state current-graph))
```

Comment:
`(state/get-file-sync-state current-graph)` is nil for non-syncing graphs.
So this becomes a `(when-not nil ... )` branch